src/enochecker/enochecker.py

```diff
@@ -419,32 +419,21 @@ def _run_method(
                 "Method {} not supported! Supported: {}".format(method, CHECKER_METHODS)
             )
 
-        ignore_run = False
+        # handle the cases where the original putflag/putnoise wasn't successful
         if method == "getflag":
             key = f"__Checker-internals-RESULT:putflag,{self.flag_round},{self.flag_idx}__"
-            if key not in self.team_db:
+            if key not in self.team_db or self.team_db[key] != "OK":
```

Comment:
the "OK" could use some explanation here, just as the original comment had.
Or maybe make it a `_putflag_failed()`

Reply:
@ldruschk ^

Reply:
There is a comment right above that: `# handle the cases where the original putflag/putnoise wasn't successful`

Reply:
And I believe the log message below that is enough explanation as well. I am not really sure what else would need explanation there